### PR TITLE
In vs out calculation for mayor, city council, and city attorney

### LINKED ISSUE #121: Create in vs out calculation for Mayor
Calculate the in vs out calculations for Mayor candidates found on https://github.com/opensandiego/sdvv-backend/blob/master/candidate_calc_description.md

In vs Out - Mayor calculation - data taken from netfile_api_2020.csv, netfile_api_2019.csv, netfile_api_2018.csv  and sd_zipcodes.csv

### PR DESCRIPTION
**Pull request by robertgz**
Fixes #121 
Resulting calculations should be verified.

### CODE REVIEW

**Comment by robertgz** [comment]
Also fixes #126

File: src/scripts/shared_routines.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+}
+
+
+function updateJSONFileWithValue( fileName, value, funWriteToObject ) {
+  const CANDIDATES_PATH = '../assets/candidates';
+
+  const filePath = `${CANDIDATES_PATH}/${fileName}`;
+
+  if ( !fs.existsSync(filePath) ) {
+    console.error(`File not found: ${filePath}`)
```
Comment (nancyheiss) [inline src/scripts/shared_routines.js:85]: Would it be better to throw an error if the file is not found?

**Review by nancyheiss — APPROVED** [review]

File: src/scripts/shared_routines.js (hunk 1 above)
Comment (robertgz) [reply src/scripts/shared_routines.js:158]: Not for the json files. There are several json files that are updated in more than one location. A missing json file may be due to a side effect of another script. The missing file should be reported out but not halt the script. Having all missing files reported out would provide more information to use when debugging the cause of the missing files.

**Comment by robertgz** [comment]
Disregard changes to the following files in this PR and instead  use PR #139 for comments on them.
> driver_script.js 
> remove_candidates_2020_folders.js 
> package.json 
> package-lock.json

**Review by nancyheiss — APPROVED** [review]

**Pull request merged**